vfcompositor/Compositor.c

```
@@ -0,0 +1,799 @@
+static TaskButton g_task_buttons[MAX_TASK_BUTTONS];
+static int g_task_button_count = 0;
+static Window* g_start_menu_window = NULL;
+
+
+typedef struct {
+    Window*             window;
+    WindowTextState     state;
+    bool                in_use;
+} WindowStateMapping;
+
+static WindowStateMapping g_window_state_map[MAX_WINDOWS];
```

Clang (14.0.6)

[warning] variable 'g_window_state_map' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)